testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/security/ClientConfigProviderTestJarHelper.java

```diff
@@ -120,13 +122,39 @@ static Process runClientConfigProviderBearerTestJar(TestType testType, String ja
     }
 
     static Process runClientConfigProviderTestJar(String jarPath, String[] args) throws IOException {
-        // use quotation marks for classpath on windows because folder names can have spaces
-        String classPath = System.getProperty("os.name").contains("indows")
-                ? "\"" + jarPath + ";" + System.getProperty("java.class.path") + "\""
+        final Pattern pattern = Pattern.compile(
+                ".*(maven|plexus|sisu|xnio|jsoup|compress|shrinkwrap|IntelliJ|mockito|arquillian|aether|wildfly|glassfish|hibernate|netty|xerces|xalan|hamcrest|io7m|slf4j|objenesis|sshd|marshalling|remoting|bytebuddy).*");
+
+        final String cp = System.getProperty("os.name").contains("indows")
+                ? jarPath + ";" + System.getProperty("java.class.path")
                 : jarPath + ":" + System.getProperty("java.class.path");
-        return Runtime.getRuntime()
-                .exec("java -cp " + classPath + " " + ClientConfigProviderTestJarHelper.PACKAGE_NAME + "."
-                        + String.join(" ", args));
+        final String[] paths = cp.split(Pattern.quote(File.pathSeparator));
+        final StringBuilder newCp = new StringBuilder();
+        for (String s : paths) {
+            final Path path = Path.of(s);
+            if (pattern.matcher(path.toString()).matches()) {
+                continue;
+            }
+            newCp.append(path).append(File.pathSeparator);
+        }
+        newCp.deleteCharAt(newCp.length() - 1);
+        // use quotation marks for classpath on windows because folder names can have spaces
+        if (System.getProperty("os.name").contains("indows")) {
```

Comment:
This should use the `TestUtil.isWindows()`. TBH we probably don't even need to test it as it's valid to surround it in quotes for Linux too.

Reply:
For some reason when the command is provided with the quotes on process builder on Linux it fails to load the class path. I know that Linux normally supports them but something is not right with the way the ProccessBuilder parses the command.

Reply:
Apparently Process Builder handles by itself when to put quotes and when not to. I removed the quotes on windows too and the tests passed, so if the tests don't fail here too it is probably ok to remove the extra quotes.